checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsAnalysis.java

```diff
@@ -18,17 +18,16 @@ public class CalledMethodsAnalysis extends AccumulationAnalysis {
    * The fully-qualified names of the exception types that are ignored by this checker when
    * computing dataflow stores.
    */
-  protected static final Set<String> ignoredExceptionTypes =
-      new HashSet<>(
-          ImmutableSet.of(
-              // Use the Nullness Checker instead.
-              NullPointerException.class.getCanonicalName(),
-              // Ignore run-time errors, which cannot be predicted statically. Doing
-              // so is unsound in the sense that they could still occur - e.g., the
-              // program could run out of memory - but if they did, the checker's
-              // results would be useless anyway.
-              Error.class.getCanonicalName(),
-              RuntimeException.class.getCanonicalName()));
+  protected static final Set<@CanonicalName String> ignoredExceptionTypes =
```

Comment:
In short: the manual section for the Called Methods Checker does not discuss ignored exceptions. We could easily change the option to `CalledMethodsChecker_ignoredExceptions`, but it would require reworking the manual a little bit.

Maybe this is worth doing, though, since the Called Methods Checker does have an undocumented default set of ignored exceptions?

Reply:
It was the latter question, and thanks for the answer.  I don't think we need a separate way to configure the Called Methods Checker ignored exceptions right now.